pkgs/development/python-modules/hfst/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, pkgs
+, stdenv
+, buildPythonPackage
+, icu
+, swig
+}:
+
+buildPythonPackage {
+  pname = "hfst";
+  inherit (pkgs.hfst) version src;
+
+  sourceRoot = "source/python";
+
+  postPatch = ''
+    # omorfi-python looks for 'hfst' Python package
+    sed -i 's/libhfst_swig/hfst/' setup.py;
+  '';
+
+  nativeBuildInputs = [
+    swig
+  ];
+
+  buildInputs = [
+    icu
+    pkgs.hfst
+  ];
+
+  setupPyBuildFlags = [
+    "--inplace"
+  ];
+
+  # Package builds but tests fail on Darwin
+  doCheck = !stdenv.isDarwin;
```

I've figured out how to fix tests on darwin:
```suggestion
  preCheck = lib.optionalString stdenv.isDarwin ''
    export DYLD_LIBRARY_PATH="${foma}/lib"
  '';
```